doc/reference/nhibernate_mapping_attributes.md

```diff
@@ -0,0 +1,371 @@
+# NHibernate.Mapping.Attributes
+
+> **NHibernate.Mapping.Attributes is an add-in for
+> [NHibernate](http://nhibernate.info/) contributed by Pierre Henri
+> Kuaté (aka *KPixel*); the former implementation was made by John
+> Morris..**
```

Comment:
Another instance of double dot. I think any `<title>` inside a `<formalpara>` has been appended with a dot. But all other cases were not already having one, so they dot not end up with a double one.

The tool you have used has likely done this because DocBook do this too, but is smarter: it adds a dot only if the title was not already ending with a dot.